DeceptionGame/Assets/Scripts/MapGeneration/MinimapController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MinimapController : MonoBehaviour
+{
+
+    public MeshFilter minimapMeshFilter;
```

Comment:
Manages displaying the minimap icons. There should be more here in the future, but for now I just hardcoded it to show the main player's location